proxy_agent/src/main.rs

```diff
@@ -1,5 +1,5 @@
 #![allow(non_snake_case)]
-#![deny(warnings)]
+//#![deny(warnings)]
```

Comment:
compile issue with authorization_rules code. functions are marked as unused even though they are being tested.

Reply:
mark it at your own new method instead.